nx-X11/lib/configure.ac

```diff
@@ -0,0 +1,283 @@
+                [Enable XLOCALEDIR environment variable support]),
+              [ENABLE_XLOCALEDIR=$enableval],[ENABLE_XLOCALEDIR=$XLOCALEDIR_IS_SAFE])
+if test "x$ENABLE_XLOCALEDIR" = "xno"; then
+        AC_DEFINE(NO_XLOCALEDIR,1,[Disable XLOCALEDIR environment variable])
+fi
+AC_MSG_RESULT($ENABLE_XLOCALEDIR)
+
+AC_ARG_ENABLE(xf86bigfont,
+              AS_HELP_STRING([--disable-xf86bigfont],
+                [Disable XF86BigFont extension support]),
+              [XF86BIGFONT=$enableval],[XF86BIGFONT="yes"])
+if test "x$XF86BIGFONT" = "xyes"; then
+# FIXME: Skip check for proto headers for now... Requires .pc files for all the proto
+# header packages we have. Keeping that in mind for later...
+#    PKG_CHECK_MODULES(BIGFONT, [xf86bigfontproto >= 1.2.0],
+        AC_DEFINE(XF86BIGFONT,1,[Enable XF86BIGFONT extension])
+#   ,XF86BIGFONT="no")
+fi
+
+AC_ARG_ENABLE(xkb,
+              AS_HELP_STRING([--disable-xkb],
+                [Disable XKB support *EXPERIMENTAL*]),
+              [XKB=$enableval],[XKB=yes])
+
+AM_CONDITIONAL(XKB, [ test x$XKB = xyes ])
+if test x"$XKB" = "xyes"; then
+   XKBPROTO_REQUIRES="kbproto"
+   X11_REQUIRES="${X11_REQUIRES} kbproto inputproto"
+   AC_DEFINE(XKB,1,[Use XKB])
+else
+   XKBPROTO_REQUIRES=""
+fi
+AC_SUBST(XKBPROTO_REQUIRES)
+
+AC_FUNC_MMAP()
+composecache_default=$ac_cv_func_mmap_fixed_mapped
+AC_CHECK_FUNC(nl_langinfo, , [composecache_default=no])
+AC_ARG_ENABLE(composecache,
+              AS_HELP_STRING([--disable-composecache],
+                [Disable compose table cache support]),
+              [COMPOSECACHE=$enableval],[COMPOSECACHE=$composecache_default])
+if test x"$COMPOSECACHE" = "xyes"; then
+        AC_DEFINE(COMPOSECACHE,1,[Include compose table cache support])
+fi
+
+X11_DATADIR="${datadir}/nx/X11"
+AX_DEFINE_DIR(X11_DATADIR, X11_DATADIR, [Location of libNX_X11 data])
+AC_SUBST(X11_DATADIR)
+
+X11_LIBDIR="${libdir}"
+AX_DEFINE_DIR(X11_LIBDIR, X11_LIBDIR, [Location of libNX_X11 library data])
+AC_SUBST(X11_LIBDIR)
+
+# FIXME: Skip check for proto headers for now... Requires .pc files for all the proto
+# header packages we have. Keeping that in mind for later...
+#PKG_CHECK_MODULES(X11, [$X11_REQUIRES])
+X11_CFLAGS="$X11_CFLAGS $XTHREAD_CFLAGS $BASE_CFLAGS"
+
+#
+# Yes, it would be nice to put the locale data in
+# /usr/share, but the locale stuff includes loadable
+# libraries which must be located in the same directory
+# as the other locale data, so for now, everything lives
+# in ${libdir}
+#
+
+X11_LOCALEDATADIR="${X11_DATADIR}/locale"
+AX_DEFINE_DIR(XLOCALEDATADIR, X11_LOCALEDATADIR, [Location of libX11 locale data])
+AC_SUBST(X11_LOCALEDATADIR)
+
+AC_ARG_WITH(locale-lib-dir,     AS_HELP_STRING([--with-locale-lib-dir=DIR],
+ [Directory where locale libraries files are installed (default: /usr/lib/locale)]),
+                                [ X11_LOCALELIBDIR="$withval" ],
+                                [ X11_LOCALELIBDIR="/usr/lib/locale" ])
```

Comment:
> Why did you replace ${X11_LIBDIR} with a hardcoded value here?

Well... because I mistook X11_LOCALELIBDIR for X11_LOCALEDATADIR. I Dropped the locale libdir path now completely, and instead hard-coded the locale data dir (+ adding a configure option for it).